src/main/resources/templates/db_scripts/setNotNullConstraintForJsonb.sql

```diff
@@ -0,0 +1,11 @@
+DO $$ BEGIN ALTER TABLE actual_opening_hours ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN ALTER TABLE exceptional_hours ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN ALTER TABLE openings ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN ALTER TABLE regular_hours ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN ALTER TABLE exceptions ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+
+DO $$ BEGIN ALTER TABLE audit_actual_opening_hours ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN ALTER TABLE audit_exceptional_hours ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN ALTER TABLE audit_openings ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN ALTER TABLE audit_regular_hours ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN ALTER TABLE audit_exceptions ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
```

Comment:
The audit_* tables changed to a new incompatible jsonb schema: https://github.com/folio-org/raml-module-builder/blob/master/doc/upgrading.md#version-26
Either delete/truncate the existing content and mention it in the release notes, or convert the content of each jsonb field to the new format.

Reply:
If I delete the existing content, do you think it is safety?

Reply:
Thanks, fixed